src/raiden_libs/matrix.py

```diff
@@ -168,12 +168,18 @@ def _start_client(self) -> None:
 
     def follow_address_presence(self, address: Address, refresh: bool = False) -> None:
         if self.user_manager:
-            log.debug("Tracking address", address=to_checksum_address(address))
             self.user_manager.add_address(address)
 
             if refresh:
                 self.user_manager.refresh_address_presence(address)
 
+            log.debug(
+                "Tracking address",
+                address=to_checksum_address(address),
+                current_presence=self.user_manager.get_address_reachability(address),
```

Comment:
Does this have a meaningful value at this time?

Reply:
At least when refresh is true it should have. But I guess this will not stay and is only to get some more infos to debug the presence problems